charts/sn-platform/conf/streamnative_console/init_instance.py

```diff
@@ -19,11 +24,17 @@
 }
 
 instanceData = {"name": os.getenv("INSTANCE_NAME"), "broker": os.getenv("WEB_SERVICE_URL")}
+serviceData = {}
 if (os.getenv("KOP_SERVICE_URL")):
-    instanceData["service"] = json.dumps({"kopServiceUrl": os.getenv("KOP_SERVICE_URL")})
+    serviceData["kopServiceUrl"] = os.getenv("KOP_SERVICE_URL")
+if (os.getenv("FLINK_SERVICE_URL")):
+    serviceData["flinkServiceUrl"] = os.getenv("FLINK_SERVICE_URL")
+instanceData["service"] = json.dumps(serviceData)
+
 instancePath = "/cloud-manager/v1/instances"
 instanceResponse = requests.post(url = host + instancePath, data = json.dumps(instanceData), headers=headers)
-if (instanceResponse.ok):
+if (instanceResponse.ok and 'Add environment success' in instanceResponse.text):
```

Comment:
It might not be ideal to rely on the message to check whether an API call succeeds. Could you give an example that `instanceResponse.ok` is true but adding environment failed?

Reply:
A health check controller will be added later